docs/dapp/tools/band.md

```diff
@@ -1,7 +1,7 @@
-# Integrating BAND oracle smart contract
+# Integrating BAND oracle
 
 This guide will explain how to query the Band Protocol reference data smart
-contract from another Solidity smart contract on Emerald.
+contract from another Solidity smart contract on an Oasis Paratime.
```

Comment:
Let's focus this chapter on Sapphire, but mention it will also run on Emerald.

```suggestion
contract from another Solidity smart contract on Oasis Sapphire, a confidential EVM-compatible chain.

:::tip

The same steps and contract addresses used in this chapter will also work
on Oasis Emerald which is an unconfidential EVM chain running on Oasis
network.

:::
```